ports/stm32f4/supervisor/port.c

```diff
@@ -35,12 +35,6 @@
 
 #include "stm32f4xx_hal.h"
 
-//#include "shared-bindings/rtc/__init__.h"
-
-static void power_warning_handler(void) {
```

Comment:
Why remove this?

Reply:
Which part? I removed the shared-bindings/rtc/__init__.h as I haven't done a deep dive into the RTC common hal yet, and I've been trying to clean up commented out material in favor of simply re-introducing it when the feature is added. `power_warning_handler` is an nrf implementation feature that I confused for a common hal component and left there, removing it shouldn't do much.